guides/common/modules/ref_operating-system-requirements.adoc

```diff
@@ -1,37 +1,47 @@
 :_mod-docs-content-type: REFERENCE
 
-[id="supported-operating-systems_{context}"]
-= Supported operating systems
+[id="operating-system-requirements"]
+= Operating system requirements
 
-ifdef::satellite[]
-You can install the operating system from a disc, local ISO image, Kickstart, or any other method that Red{nbsp}Hat supports.
-Red{nbsp}Hat {ProductName} is supported on the latest version of {RHEL} 9 that is available at the time when {ProductName} is installed.
-Previous versions of {RHEL} including EUS or z-stream are not supported.
-endif::[]
+The following operating systems are supported for deploying {Project}:
 
-The following operating systems are supported by the installer, have packages, and are tested for deploying {Project}:
-
-.Operating systems supported by {foreman-installer}
+.Operating systems supported for deploying {Project}
```

Comment:
Since you're already removing 1 column, can we adopt a simple list like in https://theforeman.org/manuals/3.15/quickstart_guide.html since it's now just 2 columns?

Reply:
I replaced the table with a list (both here and in the quickstart module).